app/views/greenTickMark.scala.html

```diff
@@ -2,6 +2,6 @@
 * This is the SVG for the green tick mark that goes before a message to indicate a successful event has occurred.
 *@
 
-<svg class="green-tick-mark" role="presentation" focusable="false" viewBox="0 0 25 25" xmlns="http://www.w3.org/2000/svg">
+<svg class="green-tick-mark" role="presentation" focusable="false" viewBox="0 0 25 25" xmlns="http://www.w3.org/2000/svg" width="25px" height="25px">
```

Comment:
Not sure styling should be directly added to the HTML, ie the dimensions?

Reply:
Usually I think you're right but the problem with it is that if it's set in the css and css is turned off, the svg icons are massive and take up most of the page so this keeps them small without changing their size on the page with css on. I'm not sure if there's a better way.

Reply:
Not sure, what is the correct way to go with this either, good practice v. good accessibility, which has greater weight. Probably needs some SVG magic to get to work properly. One for later I think.